### ISSUE TITLE
Got this error after 13 hours ago update on CE and Core_SK 

### ISSUE THREAD

**Issue by NoManColonies** [skyarkhangel/Hardcore-SK#1466]
Exception from asynchronous event: System.InvalidCastException: Cannot cast from source type to destination type.
at RimWorld.PowerConnectionMaker/<PotentialConnectorsForTransmitter>c__Iterator0.MoveNext () <0x0029a>
at RimWorld.PowerConnectionMaker.ConnectAllConnectorsToTransmitter (RimWorld.CompPower) <0x000a3>
at RimWorld.PowerNetManager.TryCreateNetAt (Verse.IntVec3) <0x00111>
at RimWorld.PowerNetManager.UpdatePowerNetsAndConnections_First () <0x00628>
at (wrapper dynamic-method) Verse.Map.FinalizeInit_Patch1 (object) <0x0007c>
at Verse.Map.FinalizeLoading () <0x00915>
at Verse.Game.LoadGame () <0x00424>
at Verse.SavedGameLoader.LoadGameFromSaveFile (string) <0x0025c>
at Verse.Root_Play.<Start>m__0 () <0x00021>
at Verse.LongEventHandler.RunEventFromAnotherThread (System.Action) <0x0002b>
Can Any body help me out? It happen when I load my save after I updated CE and Core_SK files,Thanks for having a look!!

**Comment by sidfu** [comment]
dev destory anything that useing power then use dev to rebuild them and see if it still stays

**Comment by NoManColonies** [comment]
Thanks,I will try!

**Comment by NoManColonies** [comment]
Yep,definitely fixed but still got warning string about earlier error, seem to be some furniture that use power have their property changed during update. anyway still nothing unusual happen so thanks for reply! :D

P.S. I didn't notice that stack overflow in storage had been fixed so upon I entered the game my storage explode T.T
<img width="598" alt="screen shot 2561-04-09 at 14 55 52" src="https://user-images.githubusercontent.com/38200442/38486144-57365de8-3c06-11e8-9a23-b9c48951e1c6.png">
